api/src/model/play_asset_location.gd

```diff
@@ -0,0 +1,39 @@
+# ##############################################################################
```

Comment:
I'm trying to find the best practice of comment blocks for GDScript. However, the GDScript style guide seems to have [comment styles](https://docs.godotengine.org/en/stable/getting_started/scripting/gdscript/gdscript_styleguide.html#comment-spacing) loosely set. The current comment style is adapted from [Gut](https://github.com/bitwes/Gut/blob/master/addons/gut/gut.gd).
While there are other [hacks](https://github.com/godotengine/godot/issues/21310#issuecomment-415325729) for doing multiline comment blocks in Godot, I'm still trying to figure out what is the best practice. 

Another alternative route for comment styling is to follow some Python style guides such as PEP8 or Google Python Style Guide, since the syntaxes for GDScript and Python are very similar.

Reply:
You're right that there's no particular style for GDScript comments. I think this style works, but lets ask Fredy if he knows of a better option.

Reply:
Yeah that sounds good. I will send Fredy a chat once his working hours starts.